fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1428,7 +1428,7 @@ ACTOR Future<Void> reply(CommitBatchContext* self) {
 
 	TEST(pProxyCommitData->committedVersion.get() > self->commitVersion); // later version was reported committed first
 
-	if (self->commitVersion >= pProxyCommitData->committedVersion.get()) {
+	if (self->commitVersion >= pProxyCommitData->committedVersion.get() || SERVER_KNOBS->ENABLE_VERSION_VECTOR) {
 		state Optional<std::set<Tag>> writtenTags;
 		if (SERVER_KNOBS->ENABLE_VERSION_VECTOR) {
```

Comment:
is this conditional redundant given the new one on line 1431?

Reply:
I don't think this is redundant - we can get here when version vector is disabled, right?